spec/subjects/BehaviorSubject-spec.ts

```diff
@@ -26,12 +26,27 @@ describe('BehaviorSubject', () => {
     }).to.throw(Error, 'derp');
   });
 
-  it('should throw an ObjectUnsubscribedError if getValue() is called and the BehaviorSubject has been unsubscribed', () => {
+  it('should be closed if unsubscribed', () => {
     const subject = new BehaviorSubject('hi there');
     subject.unsubscribe();
-    expect(() => {
-      subject.getValue();
-    }).to.throw(ObjectUnsubscribedError);
+    expect(subject.closed).to.be.true;
```

Comment:
What is the current behavior of calling `BehaviorSubject#getValue()` after the subject has been unsubscribed? Does it still just return the last value? Do we want to capture that in a test?